src/backend/TrafficCourtsApi/DisputeApi.Web.Test/Features/Tickets/Models/TicketModelTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using DisputeApi.Web.Features.Tickets.Models;
+using NUnit.Framework;
+
+namespace DisputeApi.Web.Test.Features.Tickets.Models
+{
+    public class TicketModelTest
+    {
+
+        [Test]
+        public void can_create_class()
+        {
+            var ticket = new Ticket { Id = "Id", Description ="Description" };
+            Assert.IsNotNull(ticket);
```

Comment:
nitpick, the new operator will never return null. Either the object will be created or an exception would be thrown.